RFC.rst

```diff
@@ -49,7 +50,6 @@ Examples of unacceptable behaviour by participants include:
   * Trolling or insulting/derogatory comments
   * Public or private harassment
   * Publishing other's private information, such as physical or electronic addresses, without explicit permission
-  * Other unethical or unprofessional conduct
```

Comment:
[My opinion about "professional"](https://github.com/kayladnls/code-manifesto/commit/d97dac251482f6b972cfab050a03935e880c7103). Not just vague—undesirable.